### DISCUSSION TITLE
How do we want to handle Ruby linting now that Rails ships with a solution?

### DISCUSSION THREAD

**Discussion by stevepolitodesign** [thoughtbot/suspenders#1307] [General]
[Rails 7.2](https://edgeguides.rubyonrails.org/7_2_release_notes.html) now ships with [rubocop-rails-omakase](https://github.com/rails/rubocop-rails-omakase).

However, with the introduction of #1148, we risk having two linting mechanisms.

**Comment by stevepolitodesign** [comment]
One solution could be to simply skip this option.

https://github.com/rails/rails/blob/ff0ef93e28d4919ffd8bf130132b7b7e3435c67a/railties/lib/rails/generators/app_base.rb#L102

**Comment by DoodlingDev** [comment]
From the notes about this change--

> If you or your team has developed a particular house style that brings you joy, you should cherish that.

`standardrb` and `standard-rails` work well, we're accustomed to them, migrating away from tbot preferences (even if it's Rails standard now) feels like the antithesis of suspenders.

Should we move to `omakase` as our recommended and most-oft-used system, then maybe we change suspenders?

**Comment by stevepolitodesign** [comment]
@DoodlingDev 

> If you or your team has developed a particular house style that brings you joy, you should cherish that.

Ha! I didn't even see that. That makes me feel good about the changes introduced in #1223

**Comment by stevepolitodesign** [comment]
The [latest release](https://github.com/thoughtbot/suspenders/releases/tag/v20251219.0) just defaults to the default settings. I did this because it will make maintenance easier, and because we can utilize the built-in [apply_rubocop_autocorrect_after_generate!](https://guides.rubyonrails.org/configuring.html#configuring-generators) feature.
